symja_android_library/matheclipse-core/src/main/java/org/matheclipse/core/expression/BigIntegerSym.java

```diff
@@ -781,9 +780,8 @@ public BigInteger toBigNumerator() {
   /** {@inheritDoc} */
   @Override
   public int toInt() throws ArithmeticException {
-    int val = NumberUtil.toIntDefault(fBigIntValue);
-    if (val != Integer.MIN_VALUE) {
-      return val;
+    if (NumberUtil.hasIntValue(fBigIntValue)) {
```

Comment:
This method could also simply call `fBigIntValue.intValueExact()`.

Reply:
yes

Reply:
Done.